config/v1/types_infrastructure.go

```diff
@@ -182,6 +182,25 @@ type BareMetalPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// provisioningInterface is the name of the network interface on a Baremetal server connected
+	// to the provisioning network.
+	ProvisioningInterface string `json:"provisioningInterface"`
+
+	// provisioningIp is the VIP to be used by the provisioning service. This must be on the provisioning
+	// subnet, and outside of the DHCP range.
+	ProvisioningIp string `json:"provisioningIp"`
+
+	// dhcpRange is the IP address range on the provisioning subnet from which Baremetal hosts
+	// can be assigned an IP address.
+	DhcpRange string `json:"dhcpRange"`
```

Comment:
@sttts I can't remember, did we default optional or required?

Reply:
Here optional https://github.com/openshift/api/blob/master/config/v1/doc.go#L5